plans/065-sink-sanitizer-gaps.md

```
@@ -0,0 +1,272 @@
+| P1       | S      | LOW  | none       | security | commit `13aa7ad0`(= `origin/main` `d3828d9e` と同内容のファイル)、2026-09-03 |
+
+## Why this matters
+
+2 つとも「解析対象リポジトリ由来の文字列を、そのシンクの文法で無害化せずに出力している」クラスで、同クラスは PR #617 / #618 / #623 で他のシンクについて閉じてきた。残っているのがこの 2 つ。
+
+### A. `@clack/prompts` の対話プロンプト
+
+CLI の端末出力は `terminalSafe`(`packages/core/src/reporter/sanitize.ts`、C0/C1/OSC/CSI を剥がす)を境界にしている。`cli-io.ts:17`、`bin.ts:16`、`install/cli.ts:26-27`(`log` / `errorLog`)、`frame-writer.ts:71` などが全てラップ済み。**例外が `@clack/prompts` に渡す文字列**で、次の 3 サイトが未適用である。
+
+1. `packages/cli/src/install/cli.ts:54` の `selectAppPrompt` — `options: apps.map((a) => ({ value: a, label: a }))`。`apps` は `discoverApps` がファイルシステムから拾った**ディレクトリ名そのまま**。
+2. `packages/cli/src/install/cli.ts:80` の `confirm` — `message: \`Apply this plan?\n${planText}\``。`planText`は`install/index.ts`の`rowLine`が`r.path` と(manual 行では)`r.snippet` を連結したもの。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)